Runtime/CSharp/Bitcraft.StateMachine.UnitTests/HandWritten/ActionTokens.cs

```diff
@@ -7,9 +7,10 @@ namespace Bitcraft.StateMachine.UnitTests.HandWritten
 {
     public static class HandWrittenActionTokens
     {
-        public static readonly ActionToken InitDoneAction = new ActionToken();
```

Comment:
Thinking out loud here but I think we better name the actions without the `Action` suffix and keep the parameterless constructor for auto naming.

The same goes for file `HandWritten/StateTokens.cs`.